_posts/2017-11-29-utdta-talk.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: tomesh @ UofT Decentralized Tech Association
+slug: decentralized tech talk
+text: The UTDTA is hosting tomesh for a talk on wireless mesh networks.
+location: Room BA1180, Bahen Centre for Information Technology, 40 St. George St  
+locationLink: http://osm.org/go/ZX6ByAqnV?
+date: 2017-11-29
+startTime: '18:00'
+endTime: '20:00'
+---
+
+Join us and the UofT Decentralized Tech Association (UTDTA) for a talk on decentralized tech and mesh networks!
+
+Guests:
+
+- Professor Peter Marbach
+- tomesh
+
+This event will be hosted by the UTDTA at {{page.startTime | date:"%-I:%M %P "}} in Room BA1180, [40 St. George Street]({{ page.locationLink }}).  
+
+All are welcome to join, just send an email to [{{ site.email }}](mailto:{{ site.email }}) so we know you are coming!
```

Comment:
Do we even need registration for this? UTDTA doesn't need it right?

Reply:
+1 that we should cut this